packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -293,15 +286,33 @@ describe('Remove item from cart', function () {
         )
     })
     test('Can remove item from the cart', async () => {
+        if (process.env.CI) {
+            console.log('--- skipping this test only in CI because it has proven to be flaky there')
```

Comment:
This cart test is annoyingly flaky. I couldn't figure out why a simple `findBy` query can lead to a timeout. Have tried various things. 

Anyways, I decided to conditionally skip this test if it runs on CI.

Reply:
I think I'd rather we just `test.skip` it than execute it conditionally on CI

Reply:
Ok, I'll skip the test.